js/main.js

```diff
@@ -0,0 +1,67 @@
+// navbar
+const menuBtn = document.querySelector('.menu-btn');
+const burger = document.querySelector('.burger');
+const header = document.querySelector('header');
+const menuNav = document.querySelector('.menu-navbar');
+let displayMenu = false;
+
+function toggleMenu() {
+  if (!displayMenu) {
+    burger.classList.add('open');
+    header.classList.add('open');
+    header.classList.add('open');
+    menuNav.classList.add('open');
+    displayMenu = true;
+  } else {
+    burger.classList.remove('open');
+    header.classList.remove('open');
+    menuNav.classList.remove('open');
+    displayMenu = false;
+  }
+}
+
+menuBtn.addEventListener('click', toggleMenu);
+
+// feature speakers
+
+const speakers = [
+  {
+    id: '01',
+    name: 'Yochai Benkler',
+    title: 'Lorem ipsum dolor sit amet.',
+    paragraph: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt, nisi esse.',
+    img: 'assets/img1.jpg',
+  },
+  {
+    id: '02',
+    name: 'Yochai Benkler',
+    title: 'Lorem ipsum dolor sit amet.',
+    paragraph: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt, nisi esse.',
+    img: 'assets/img2.jpg',
+  },
+];
```

Comment:
- [Optional] Kindly, It would be better if you add more speakers to make it more engaging and stick to the given design as the previous reviewer suggested you 👌